native_client/javascript/Makefile

```diff
@@ -40,7 +40,7 @@ copy-deps: build
 node-wrapper: copy-deps build
 	$(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_ABI_TARGET) package $(NODE_BUILD_VERBOSE)
 
-npm-pack: clean package.json README.md
+npm-pack: clean package.json README.md index.js
```

Comment:
I don't see anything being generated in `index.js`: am I missing something, or should we remove that from the npm-pack target?

Reply:
This is so that (manual) changes to that file invalidate the `npm-pack` target.